netlify.toml

```diff
@@ -3,7 +3,7 @@ command = "echo 'no op'"
 publish = "example"
 
 [build.environment]
-NODE_VERSION = "16.15.1"
+NODE_VERSION = "17.9.1"
```

Comment:
Let's stay on LTS

Reply:
we don't have an LTS on `v17.*` so I set it to `16.15.1` which is an LTS